pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -60,6 +63,28 @@ def __init__(
         self.df_whitelist = load_data.load_whitelist()
         self.df_whitelist = self.df_whitelist[self.df_whitelist["inference_ok"] == True]
 
+    @staticmethod
+    def _get_county_hospitalization_from_actuals(fips, t0_simulation):
+        county_hosp = load_data.get_current_hospitalized_for_county(
+            fips, t0_simulation, category="hospitalized"
+        )[1]
+        county_icu = load_data.get_current_hospitalized_for_county(
+            fips, t0_simulation, category="icu"
+        )[1]
+
+        return county_hosp, county_icu
+
+    @staticmethod
+    def _is_valid_count_metric(metric: float) -> bool:
+        return metric is not None and metric > 0
+
+    def _get_population(self, fips: str):
+        if len(fips) == 5:
+            return self.population_data.get_record_for_fips(fips)[CommonFields.POPULATION]
+        return self.population_data.get_record_for_state(self.state_abbreviation)[
+            CommonFields.POPULATION
+        ]
+
     def map_fips(self, fips):
         """
         For a given county fips code, generate the CAN UI output format.
```

Comment:
the implementation looks like it may handle a 2 char state fips code. please clarify the docstring while you are deep in the code.

Reply:
good catch, I didn't even notice this doc string, but it's clearly wrong.